minari/utils.py

```diff
@@ -567,7 +566,7 @@ def create_dataset_from_buffers(
 
 def create_dataset_from_collector_env(
     dataset_id: str,
-    collector_env: DataCollectorV0,
+    collector_env: Any,
```

Comment:
to avoid circular import,  in DataCollectorV0, you can import `from minari.utils import _generate_dataset_metadata, _generate_dataset_path` inside the `create_dataset` function (please also add a #TODO comment saying to move it on top with 0.5.0 release)

Reply:
This change is also done.